Autopilot/Path Manager/ProbeDrop.h

```diff
@@ -0,0 +1,13 @@
+/* 
+ * File:   ProbeDrop.h
+ * Author: banky
```

Comment:
If you are going to put in author docs, I recommend putting in ones that people can actually find you with. IE. Full name and/or email.